e3sm_diags/driver/utils/dataset_xr.py

```diff
@@ -389,7 +389,9 @@ def get_climo_dataset(self, var: str, season: ClimoFreq) -> xr.Dataset:
 
         if self.is_time_series:
             ds = self.get_time_series_dataset(var)
+
             ds_climo = climo(ds, self.var, season).to_dataset()
+            ds_climo = ds_climo.bounds.add_missing_bounds(axes=["X", "Y"])
```

Comment:
This should close #884. Please verify the fix, thanks.

Reply:
Indeed, this fixed the problem! Thank you!